### PR TITLE
add new game button

### PR DESCRIPTION
**Pull request by DusanSacha**
Implemented #35 

I switched off linebreak-style rule in  eslintConfig. The reason for this was that in my VSCode I had on each line a warning that I should change the line ending (I am on windows). It was kind of disturbing and made me easily overlook other linting errors. I think it doesn't affect our coding.

### CODE REVIEW

File: src/components/Menu.js [hunk 1]
```diff
@@ -65,19 +67,28 @@ class Menu extends Component {
         <ToolbarGroup>
           <RaisedButton
             className="menuButton"
+            label="New game"
+            onTouchTap={onNewClick}
+            title="Start a new game with random numbers order."
```
Comment (ovidiubute) [inline src/components/Menu.js]: I think something like "Start a new game" is good enough to be honest.

File: src/components/Menu.js [hunk 2]
```diff
@@ -65,19 +67,28 @@ class Menu extends Component {
...
                 : <Pause className="menuIcon" />
             }
+            title="Pause/Continue current game."
             disabled={gameState !== GAME_STARTED && gameState !== GAME_PAUSED}
           />
           <RaisedButton
             className="menuButton"
             label="Reset game"
             onTouchTap={onResetClick}
+            title="Reset current game to original numbers order."
```
Comment (ovidiubute) [inline src/components/Menu.js]: "Reset game", keep it short and sweet

**Comment by ovidiubute** [comment]
Feel free to merge this yourself when ready.

**Comment by DusanSacha** [comment]
Thanks for the review. I will merge it now.

**Pull request merged**